src/Chapter2.hs

```diff
@@ -621,7 +637,10 @@ Write a function that takes elements of a list only in even positions.
 >>> takeEven [2, 1, 3, 5, 4]
 [2,3,4]
 -}
-takeEven = error "takeEven: Not implemented!"
+takeEven :: [a] -> [a]
+takeEven [] = []
+takeEven (x:xs) = x : takeEven (drop 1 xs)
```

Comment:
Awesome! 💯 
Or you can explicitly pattern match on that, so you won't need to call `drop` function:

```suggestion
takeEven (x:_:xs) = x : takeEven xs
takeEven [x] = [x]
```